PayPal/src/main/java/com/braintreepayments/api/paypal/PayPalClient.java

```diff
@@ -281,6 +286,23 @@ public void tokenize(@NonNull PayPalPaymentAuthResult.Success paymentAuthResult,
         }
     }
 
+    @ExperimentalBetaApi
+    public void createEditAuthRequest(
```

Comment:
Do we want to go ahead and add docstrings for the public method?

Reply:
Yep, I forgot to add them! I just pushed the changes.